editor.planx.uk/src/@planx/components/shared/types.tsx

```diff
@@ -6,6 +6,7 @@ export interface EditorProps<Type, Data> {
   id?: string;
   handleSubmit?: (data: { type: Type; data: Data }) => void;
   node?: any;
+  disabled?: boolean;
```

Comment:
```suggestion
  disabled: boolean;
```

We'll eventually want to make this mandatory. Having it optional for now allows you to fix a component at a time however. Something to make required just before opening for review I think 👍

Reply:
Just seen this comment applies a few times throughout this PR!